Core/UI/steering/racing_tabs/tab_debug.h

```diff
@@ -31,6 +31,11 @@ typedef enum {
   TAB_LV_ERRORS
 } ErrorTabID;
 
+extern lv_obj_t *hv_feedbacks_status[20];
```

Comment:
Instead of declaring these extern you should define the update function in this file instead of steering.c. This will simplify design and also it will be the same as the other tabs